UnattendResources/Logon.ps1

```diff
@@ -214,6 +214,46 @@ function License-Windows {
     }
 }
 
+function Get-AdministratorAccount {
+    <#
+    .SYNOPSIS
+    Helper function to return the local Administrator account name. This works with internationalized versions of Windows.
+    #>
+    PROCESS {
+        $version = $PSVersionTable.PSVersion.Major
+        if ($version -lt 4) {
+            # Get-CimInstance is not supported on powershell versions earlier then 4
+            New-Alias -Name Get-ManagementObject -Value Get-WmiObject
+        } else {
+            New-Alias -Name Get-ManagementObject -Value Get-CimInstance
+        }
+        $SID = "S-1-5-21-%-500"
+        $modifier = " LIKE "
+        $query = ("SID{0}'{1}'" -f @($modifier, $SID))
+        $s = Get-ManagementObject -Class Win32_UserAccount -Filter $query
+        if (!$s) {
+            Throw "SID not found: $SID"
+        }
+        return $s.Name
+    }
+}
+
+function Enable-Administrator {
+    $username = Get-AdministratorAccount
+    $setupCompletePath = "$env:windir\Setup\Scripts\SetupComplete.cmd"
+    $activate = "powershell -c net user Administrator /active:yes"
+    $expiration = 'wmic path Win32_UserAccount WHERE Name="Administrator" set PasswordExpires=true'
+    $logonReset = "net.exe user Administrator /logonpasswordchg:yes"
+    Add-Content -Encoding Ascii -Value $activate -Path $setupCompletePath
+    Add-Content -Encoding Ascii -Value $expiration -Path $setupCompletePath
+    Add-Content -Encoding Ascii -Value $logonReset -Path $setupCompletePath
+    $command = "net.exe user $username """
+    & cmd.exe /c $command

```

Comment:
put here a note

Note(atira): net.exe can set an empty password only if it is run from cmd.exe